recipes/openmm-atmmetaforce-plugin/meta.yaml

```
@@ -0,0 +1,68 @@
+{% set name = "openmm-atmmetaforce-plugin" %}
+{% set version = "0.2.4.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Gallicchio-Lab/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: cf7975639c285ed0fab441c2ac36e3cbf6e499256a708240d4acfeb562939bae
+
+build:
+  number: 0
+  skip: true  # [not linux64]
+  skip: true  # [cuda_compiler_version in (undefined, 'None')]
+
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}  # [linux]
+    - cmake
+    - swig
+
+  host:
+    - python
+    - openmm
+    - pthread-stubs
+    - lxml
+    # OpenCL ICD
+    - ocl-icd  # [linux]
+    - khronos-opencl-icd-loader  # [win or (osx and opencl_impl == 'khronos')]
+    
+  run:
+    - python
+    - openmm
+
+test:
+  requires:
+    - pocl  # [unix]
+    - pip
+  imports:
+    - simtk
+    - openmm
+  commands:
+    - pip check
+    - test -f $PREFIX/lib/plugins/libATMMetaForcePluginCUDA$SHLIB_EXT  # [unix]
+    - test -f $PREFIX/lib/plugins/libATMMetaForcePluginOpenCL$SHLIB_EXT  # [unix]
+    - test -f $PREFIX/lib/plugins/libATMMetaForcePluginReference$SHLIB_EXT  # [unix]
```

```suggestion
  commands:
    - pip check
    - test -f $PREFIX/lib/plugins/libATMMetaForcePluginCUDA$SHLIB_EXT  # [unix]
    - test -f $PREFIX/lib/plugins/libATMMetaForcePluginOpenCL$SHLIB_EXT  # [unix]
    - test -f $PREFIX/lib/plugins/libATMMetaForcePluginReference$SHLIB_EXT  # [unix]
    - test -f $PREFIX/include/ATMMetaForce.h  # [unix]
    - test -f $PREFIX/include/internal/ATMMetaForceImpl.h  # [unix]
```
Test the existence of a header since you intend to ship them. Helps catch moving or renaming of headers at a new release.